calculation/gmhazard_calc/gmhazard_calc/dbs/SiteSourceDB.py

```diff
@@ -83,6 +83,21 @@ def has_station_data(self, station_name):
 
         return key in self._keys_cache
 
+    @check_open
+    def has_station_directivity_data(self, station_name):
```

Comment:
How come this is in this PR? Doesn't really have anything to do with spatial hazard?